.agents/plans/mcp-inspector.md

```
@@ -0,0 +1,317 @@
+expect(stripSnapshot('<path d="M 0 0 L 100 100"/>')).toBe('<path … />')
+
+// config generation
+expect(generateConfig({ preset: { value: 'rdg', confidence: 0.95 }, ... }))
+  .toContain('...Plugins.RDG.config()')
+```
+
+### Integration tests (Playwright, fixture pages)
+
+Static HTML fixtures in `packages/mcp/tests/fixtures/` representing each detectable case — no framework, just hand-crafted HTML with the right signals. A simple `http.createServer` serves them during the test run.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)